src/System.Runtime/samples/DateTime/DateTime.cs

```diff
@@ -0,0 +1,54 @@
+using System;
+
+public class Program
+{
+	public static void Main()
+	{
+		// Demonstrates common uses of DateTime struct.
+		// Doc: https://msdn.microsoft.com/library/system.datetime.aspx
+		
+		// Get and manipulate time
+		Console.WriteLine("Print local DateTime");
+		Console.WriteLine(DateTime.Now);
+		Console.WriteLine();
+		
+		Console.WriteLine("Print UTC DateTime");
+		Console.WriteLine(DateTime.UtcNow);
+		Console.WriteLine();
+		
+		Console.WriteLine("Print local DateTime + 10 mins");
+		Console.WriteLine(DateTime.Now.AddMinutes(10));
+		Console.WriteLine();
+		
+		Console.WriteLine("Print local DateTime - 5 days and 4 hours");
+		Console.WriteLine(DateTime.Now.AddDays(-5).AddHours(-4));
```

Comment:
Echoing @stephentoub's comment - I've seen too many people get bit by math on local `DateTime` values.  We shouldn't perpetuate bad practices in our examples.